### ISSUE TITLE
Background vs text contrast

### ISSUE THREAD

**Issue by andrewckinstler** [andrewckinstler/fitlit-refactor#7]
Failing Elements
--
span#user-current-step-count-vs-average
span#all-users-average-step-count
span#user-current-active-mins
span#all-users-average-active-mins
span#user-current-stairs-climbed
span#all-users-average-stairs-climbed
p#winner-name
